gnomad_qc/v3/create_release/create_hgdp_tgp_subset.py

```diff
@@ -45,83 +52,106 @@
 logger.setLevel(logging.INFO)
 
 AS_FIELDS.remove("InbreedingCoeff")
-SITE_FIELDS.remove("BaseQRankSum")
 
-GLOBAL_SAMPLE_ANNOTATION_DICT = {
-    "sex_imputation_ploidy_cutoffs": {
+GLOBAL_SAMPLE_ANNOTATIONS = {
```

Comment:
Yeah I did think about it, but wasn't sure which was best since these would be the only things couple constants in that file. However, we do this for the resources constants too, so I will change. Do you think just a `gnomad.v3.create_release.constants` or a `gnomad.v3.create_release.hgdp_tgp_constants` since these are specific to the hgdp_tgp subset?

Reply:
I think the latter sounds great!